### DISCUSSION TITLE
Compatibility with a BBOS

### DISCUSSION THREAD

**Discussion by LukkyLK** [JoseskVolpe/UndertaleME#5] [Q&A]
Hi! , So, I was thinking if you want some help to test the Midlet 
:3, I have an Blackberry Curve 8520.

320x240 Screen.
Running the 5.0.0.xxx Platform/Os

And, I am having some problems like... the .jar and .jad files don't work on the Kemulator (that I use to test)
you have any clue ?

**Comment by JoseskVolpe** [comment]
Just a moment.... Let me check the lastest .jar files

**Comment by JoseskVolpe** [comment]
Yeah, that's weird. It isn't working in microemulator and it freeze in WTK. I'm not finished testing it, it might be a bug in my WTK because i'm still configuring it (i'm in a new operating system i just installed now), i need some more time to try it

**Reply by LukkyLK** [reply]
Oh, i have 0% Knowledge in Java , so...
I hope you find a way to fix it 
:³ good luck dude

**Comment by JoseskVolpe** [comment]
I just tested it on KEmulator, and it's working
![Screenshot_20210813_003511](https://user-images.githubusercontent.com/78318343/129435245-a7089a28-7958-4699-9a0e-728d2afee532.png)

I think you're trying to load it from JAD... KEmulator is very glitchy to work with JAD files, try loading it from JAR instead

Also, you may send me logs for your blackberry phones too ;3

**Reply by LukkyLK** [reply]
Well it's better if I say it in portuguese.

O problema é que... Quando eu tento compilar no Eclipse, ele me vários erros porém eu tentei abrir outros projetos demos e eles também estão dando erros.

Não, eu nunca usei o Eclipse :D

**Reply by JoseskVolpe** [reply]
Bem, não me lembro exatamente como que tá o estado do código desde a última commit jjjasjjjas
Mas acho que o código da última commit pode, talvez, estar incompilável ainda, porque tem uma parte incompleta
Mas não tenho certeza, tenho que ver isso

¿Ele retorna que erro na compilação?

**Reply by JoseskVolpe** [reply]
Você pode encontrar o jar compilado aqui: https://github.com/JoseskVolpe/UndertaleME/tree/main/.mtj.tmp/emulation

**Reply by LukkyLK** [reply]
> Bem, não me lembro exatamente como que tá o estado do código desde a última commit jjjasjjjas
> Mas acho que o código da última commit pode, talvez, estar incompilável ainda, porque tem uma parte incompleta
> Mas não tenho certeza, tenho que ver isso
> 
> ¿Ele retorna que erro na compilação?

- Ele retorna erro ao carregar o projeto, se quiser eu posso mandar uma foto amanhã do Eclipse pra ver se você sabe o que está acontecendo :,)

**Reply by JoseskVolpe** [reply]
blz

**Reply by LukkyLK** [reply]
E sobre o yiff... .w. melhor nao kkkk
Não quero te dar traumas, btw tu conhece um tal de Nytro? Uwu

**Reply by JoseskVolpe** [reply]
¿Pq eu ficaria traumatizado? 😹😹😹😹😹

Conheço

**Reply by LukkyLK** [reply]
Entao o nytro te conhece mesmu :0

Btw, sobre o yiff, é complicado quando você não sabe desenhar e tenta :D

**Reply by JoseskVolpe** [reply]
Sim, conheço :3
Ele tava falando de um tal amigo dele que queria falar comigo, ¿é vc?

E nah, não me importo com o estilo uwu
Aquela pré-visualização nem parece ruim tbm

**Reply by JoseskVolpe** [reply]
Você disse que não conhecia Java... ¿Vc conhece alguma linguagem de programação, tipo Python?
To pretendendo  integrar o Hecl no UndertaleME. Daí a linguagem de programação que pode ser usada para criar batalhas vai ser o [Hecl](https://www.hecl.org)
Ele lembra bastante o Python e o Shell

https://github.com/davidw/hecl

**Reply by LukkyLK** [reply]
Bem, eu sei o básico de Python.
E esse Hecl acho que já ouvi falar

**Reply by JoseskVolpe** [reply]
Que coincidência rsrsrs
Essa linguagem foi muito pouco usada

**Comment by JoseskVolpe** [comment]
Also... I would like to see the full version of your last art on Twitter alongside the logs too... Weeeeelllll.. That's for.. debugging reasons... hehe ">w<
But that's optional >w<


lol 😹😹😹 💧😸

**Reply by LukkyLK** [reply]
pwp eu nunca posto artes no twitter, prefiro o insta.

https://www.instagram.com/p/CJJlff0pm9g/?utm_medium=copy_link

**Reply by JoseskVolpe** [reply]
Bem.. Não uso Instagram ajsjasj

E... eeerrr... A Última arte do Twitter sua, quero dizer, é um yiff >w>
